src/ui-components/custom-overflow-menu/custom-items/edit-patient-details.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import CustomOverflowMenuItem from "../overflow-menu-item.component";
+import { useTranslation } from "react-i18next";
+
+export default function EditPatientDetailsOverflowMenuItem() {
+  const { t } = useTranslation();
+  return (
+    <CustomOverflowMenuItem
+      itemText={t("Edit Patient Details", "Edit Patient Details")}
+    />
+  );
+}
```

Comment:
You can just delete this file

Reply:
I thought this can be used later. If you are saying I'll delete the file as well.

Reply:
I have done some changes in this file on my pc. I have linked it to /edit page of registration. So when the Patient registration PR will be merged, I'll push the new changes.

Reply:
Let's do one thing at a time. You can always open a new PR. In fact, smaller PRs are better.

Reply:
Ohh correct. Sorry about that.

Reply:
Done